src/main/java/forestry/arboriculture/tiles/TileLeaves.java

```diff
@@ -87,6 +93,10 @@
 
 	private IEffectData[] effectData = new IEffectData[2];
 
+	public TileLeaves() {
+		super(ModuleArboriculture.getTiles().leaves);
+	}
+
 	/**
 	 * Worldgen trees used to create TileLeaves, but now they use BlockDefaultLeaves instead.
 	 * We add a check to convert the TileLeaves into the new format.
```

Comment:
Can we remove this check now?

Reply:
I totally forgot about that. You are right.